Library/Homebrew/test/dev-cmd/bottle_spec.rb

```diff
@@ -194,3 +194,104 @@ def stub_hash(parameters)
     )
   end
 end
+
+describe "brew bottle --merge", :integration_test do
```

Comment:
Go for `:needs_linux` for this one because the macOS tests are so slow already and this will make them a bit slower.

Reply:
Not resolved yet.

Reply:
Sorry I forgot to add it, I was debugging this on Mac. Done